src/main/java/net/irisshaders/batchedentityrendering/impl/ordering/GraphTranslucencyRenderOrderManager.java

```diff
@@ -46,10 +48,21 @@ private static TransparencyType getTransparencyType(RenderType type) {
 	}
 
 	public void begin(RenderType renderType) {
-		TransparencyType transparencyType = getTransparencyType(renderType);
+
+		if (processedRenderTypes.contains(renderType)) {
+			return;
+		}
+
+		TransparencyType transparencyType = translucentRendered ? TransparencyType.GENERAL_TRANSPARENT : getTransparencyType(renderType);
```

Comment:
Would probably be beneficial to add a small comment explaining why this is necessary for future readers.